diff --git a/node_modules/@tanstack/start-config/dist/esm/vinxi-file-router.js b/node_modules/@tanstack/start-config/dist/esm/vinxi-file-router.js
index 68e9df1..2869c61 100644
--- a/node_modules/@tanstack/start-config/dist/esm/vinxi-file-router.js
+++ b/node_modules/@tanstack/start-config/dist/esm/vinxi-file-router.js
@@ -1,5 +1,5 @@
 import { BaseFileSystemRouter as VinxiBaseFileSystemRouter, analyzeModule as vinxiFsRouterAnalyzeModule, cleanPath as vinxiFsRouterCleanPath, } from 'vinxi/fs-router';
-import { CONSTANTS as GENERATOR_CONSTANTS, startAPIRouteSegmentsFromTSRFilePath, } from '@tanstack/router-generator';
+import { CONSTANTS as GENERATOR_CONSTANTS, startAPIRouteSegmentsFromTSRFilePath, } from '@tanstack/router-generator/dist/esm/generator.js';
 export function tanstackStartVinxiFileRouter(opts) {
     const apiBaseSegment = opts.apiBase.split('/').filter(Boolean).join('/');
     const isAPIPath = new RegExp(`/${apiBaseSegment}/`);
